src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,10 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+To make a request, you need to specify the **tag name**, the **Id of the chat** and the **Id of the thread** you want to add a **tag** to. Active and inactive chats can be tagged, and one or more tags can be added to the specific **thread**. To tag a chat, the **tag** needs to be created first. It's possible to do that in the [LiveChat Application](https://accounts.livechat.com/) or via the [Configuration API](/management/configuration-api). Tags are case sensitive and are defined by each client individually. The **list of tags** can be found in the [LiveChat Application](https://my.livechatinc.com/settings/tags). 
```

Comment:
```suggestion
Active and inactive chats can be tagged, and one or more tags can be added to the specific **thread**. To tag a chat, the **tag** needs to be created first. It's possible to do that in the [LiveChat Application](https://accounts.livechat.com/) or via the [Configuration API](/management/configuration-api). Tags are case sensitive and are defined by each client individually. The **list of tags** can be found in the [LiveChat Application](https://my.livechatinc.com/settings/tags). 
```

Reply:
We could shorten the description by deleting the first sentence. The same information is presented in the table below (request parameters). I suggest starting the description with a simple statement, for example:

> Tags a given thread.
>
> The rest of the description.